### PR TITLE
development mode

### PR DESCRIPTION
**Pull request by sumew**
This PR includes a couple of changes that I made testing my changes and playing around with the project a bit easer. Please note that I have limited front-end experience and am not familiar with zephyr so if there's a better way to run the build in watch mode that doesn't involve copying files, I'd love to make the appropriate changes.

- Copied `index.js` to `index-dev.js` and made it point to `./output/Main` instead of `./dce-output/Main`
- Copied the `assets` folder to `dev` and made `index.html` point to the `index-dev.js`
- Added instructions for running in dev mode

### CODE REVIEW

**Comment by thomashoneyman** [comment]
I don't use hot-reloading myself, so I'm not completely sure about the best way to put this together. @milesfrain put together the [Halogen starter template](https://github.com/purescript-halogen/purescript-halogen-template) which does have it (and could serve as an example) -- do you have any suggestions?

File: README.md [hunk 1]
```diff
@@ -55,6 +57,20 @@ And, once bundled, you can run a local server to use Conduit (defaults to [port
 ```sh
 npm run serve
 ```
+## Dev mode
+
+To run it in dev mode where saving your changes rebuilds and reloads the app, you can run the command below (which calls `spago build --watch`) 
+
+```sh
+npm run watch
+```
```
Comment (milesfrain) [inline README.md:66]: This step should be unnecessary if using an IDE, such as vscode+PS extension, since it already recompiles upon save.

**Comment by milesfrain** [comment]
This looks good.

There's an annoying tradeoff to make though with any solution, and that's whether to:
* Make a duplicate `index.html` (as is done in this PR). That means that now there are two essentially identical files to maintain.
* Use the same `index.html`, but overwrite the contents of its .js entry point. This requires some file manipulations in the launch scripts, and might behave in strange ways if you have a dev server running while simultaneously creating a production bundle.

I'm eagerly awaiting ES modules, since I think that will simplify a lot of these bundling and hot-reload steps.

**Review by thomashoneyman — APPROVED** [review]

**Comment by thomashoneyman** [comment]
Thanks!

**Pull request merged**